Makefile

```diff
@@ -51,4 +51,5 @@ production:
 cf-push:
 	$(if ${CF_SPACE},,$(error Must specify CF_SPACE))
 	cf target -s ${CF_SPACE}
+	cf unbind-service notify-paas-autoscaler notify-db
 	cf push -f <(make generate-manifest)
```

Comment:
Do we need to bind the apps again afterwards?

Reply:
No, it's defined in [the manifest](https://github.com/alphagov/notifications-paas-autoscaler/blob/master/manifest.yml.erb#L27) that this app requires these services. When we `make cf-push` it takes the manifest into account.